pktfwd/manager.go

```diff
@@ -233,16 +233,25 @@ func (m *Manager) statusRoutine(bgCtx context.Context, errC chan error) {
 	}
 }
 
+func (m *Manager) networkRoutine(bgCtx context.Context, errC chan error) {
+	err := m.netClient.RefreshRoutine(bgCtx)
+	if err != nil {
+		errC <- errors.Wrap(err, "Couldn't refresh account server token")
+	}
+}
+
 func (m *Manager) startRoutines(bgCtx context.Context, err chan error, runTime time.Time) {
-	var errC = make(chan error)
+	var errC = make(chan error, 4)
```

Comment:
Isn't it cleaner for the methods to return their own error channel? This results in multiple error channels, but at least the error owner (i.e. the function) controls the capacity, whether or not to block and close the channel. This is also safer against sending after closing, as the function closes it